### PR TITLE
Deprecate in favor of ItemVersion

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
FYI @martin-brajer, I think this is the way forward. Thank you for taking over though :)

### CODE REVIEW

**Review by ah0tk3PS9m — COMMENTED** [review]
Not so short, but yea - less overlapping addons is better :)

File: README.rst [hunk 1]
```diff
@@ -3,3 +3,8 @@ wow-item-expansion
 
 Displays the item's extension of origin in item's tooltip; so you can see easily from which expansion an object
 originate from.
+
+:warning: Deprecated in favor of https://www.curseforge.com/wow/addons/itemversion. And inferior.
```
Comment (ah0tk3PS9m) [inline README.rst:7]: Is the "And inferior." part intended? It seems as residue from some previous version.

Comment (Pierre-Sassoulas) [reply README.rst:7]: My syntax. Bad. 😄

**Comment by Pierre-Sassoulas** [comment]
Also upgraded the description at : https://www.curseforge.com/wow/addons/items-expansions-in-tooltip

The new screenshot was nice :)

**Pull request merged**